jwt/api_jws.py

```diff
@@ -101,7 +103,11 @@ def encode(self, payload, key, algorithm='HS256', headers=None,
             signature = alg_obj.sign(signing_input, key)
 
         except KeyError:
-            raise NotImplementedError('Algorithm not supported')
+            if not has_crypto and algorithm in get_crypto_algorithms():
+                raise NotImplementedError('"cryptography" package must be '
+                                          'installed to use this algorithm')
```

Comment:
How about something like `Algorithm %s could not be found. Do you have cryptography installed?`